src/learning_item.py

```diff
@@ -0,0 +1,28 @@
+class LearningItem:
+    def __init__(self, id, is_taught, is_verb_irregular, questions, answers,
+                 interval, repetition_date, easiness_factor, number_of_correct_repetitions,
+                 number_of_incorrect_repetitions):
+        self.id = id
+        self.is_taught = is_taught
+        self.is_verb_irregular = is_verb_irregular
+        self.questions = questions
+        self.answers = answers
+        self.interval = interval
+        self.repetition_date = repetition_date
+        self.easiness_factor = easiness_factor
+        self.number_of_correct_repetitions = number_of_correct_repetitions
+        self.number_of_incorrect_repetitions = number_of_incorrect_repetitions
+
+    def to_dict(self):
```

Comment:
i found a method that is built in every object so you can use it instead of this one. I cannot write the name of it because github is just formating the text inside as bold so im going to paste a screenshot

Reply:
![image](https://github.com/PremGorecki/memoryBooster/assets/50406287/b30f8c28-f376-4902-aed5-1502a11fd7ee)